frontend/src/components/Town/interactables/Chess/ChessArea.tsx

```diff
@@ -122,10 +122,14 @@ function ChessArea({ interactableID }: { interactableID: InteractableID }): JSX.
   const multiplayerPage = () => {
     setcurrentPage('multiplayer');
   };
+  const gameReviewPage = () => {
+    setcurrentPage('gamereview');
+  };
+
   useEffect(() => {
-    // Fetch chess game results (for now, using dummy data)
-    const results = generateDummyChessResults();
-    setChessResults(results);
+    fetchAllGames().then((fetchedGames: React.SetStateAction<GameData[]>) => {
```

Comment:
The current flow seems to allow people to review all past games, but we only want show show games they have played in. I think this can be fixed by doing a filter here on check if either white or black is our player.

Reply:
So I don';t think this will work currently for a few reasons but i do think there are workarounds we could do. 
1. If there are more then 2 players, white or black check doesn't work
2. We could go by usernames but I dont think we have anything setup to check this currently
3. I think this is an overall good solution until we have more of the code base set up.